tests/onnxruntime/test_modeling_ort.py

```diff
@@ -286,15 +290,15 @@ def test_supported_transformers_architectures(self, *args, **kwargs):
 
     def test_load_vanilla_transformers_which_is_not_supported(self):
         with self.assertRaises(Exception) as context:
-            model = ORTModelForTokenClassification.from_pretrained("t5-small", from_transformers=Tru)
```

Comment:
the idea was here to use a "architecture" which we don't support with `from_transformers`

Reply:
Yep you're right, this is not the change I wanted to push actually XD
I wanted to add a "e" at `from_transformers=Tru`, I'm going to push this in a new commit.

Reply:
Thinking about it a bit more, I should actually remove `from_transformers=True` here right @philschmid ? Or setting it to `False`

Reply:
Yes!

Reply:
Okay I understand better. There was a bug in all the `test_load_vanilla_transformers_which_is_not_supported`. Since it was `from_transformers=Tru` and not `from_transformers=True`, an exception was raised as expected. However, then, `self.assertTrue("Unrecognized configuration class", context.exception)` will always pass because it assesses the truthness of the 1st argument (`"Unrecognized configuration class"` here). I guess what we want to do here is to check if `"Unrecognized configuration class"` is in `context.exception`. I'm going to fix this.

Reply:
Also, for the feature extraction task, all models are supported (see [here](https://github.com/huggingface/transformers/blob/f47afefb210a120545a5e9745292aaf6c316d246/src/transformers/pipelines/feature_extraction.py#L15)) so I will just remove this test for this task.